src/ir/enum_ty.rs

```diff
@@ -71,16 +71,20 @@ impl Enum {
         };
 
         declaration.visit(|cursor| {
-            if cursor.kind() == CXCursor_EnumConstantDecl {
```

Comment:
I don't think there's a runtime penalty for using `unwrap` vs `if let`. Maybe rust annotates `unwrap` as `unlikely` so it has a bit better branch prediction, but nothing measurable for this case I think :)